src/wp-admin/css/forms.css

```diff
@@ -550,7 +550,53 @@ fieldset label,
 }
 
 .wp-pwd {
-	margin-top: 1em;
+	display: inline-block;
```

Comment:
I've fixed the spanning issue for narrower screen sizes. However, I also noticed the password manager icons are overlapping the toggle button. When I have more time I'll attempt to address this as well.

![Screenshot_2023-01-05_22-49](https://user-images.githubusercontent.com/15313658/210932370-dd5687c8-f7fa-4e78-b352-627cd782ca3e.png)

Reply:
In an attempt to fix the overlap issue I commented on, as well as the incorrect lang support [I added here](https://github.com/WordPress/wordpress-develop/pull/3725/commits/31210f927c70885775c7b80df56fba0ca54ec3a9), I noticed that the profile page was hiding the pw toggle text on narrower screens. So I added `<span class="text"><?php _e( 'Show' ); ?></span>` to test. Unfortunately, this creates white space that is not taken up by the pw field or pw manager icons and the button is still focusable. It may be better to hide the button entirely and re-add the 10px padding that is added to input fields [here](https://core.trac.wordpress.org/browser/tags/6.1/src/wp-admin/css/install.css#L283) for narrower screens.

![Screenshot_2023-01-05_23-56](https://user-images.githubusercontent.com/15313658/210939512-5cb3d4d1-9b22-4e21-8c72-502a1467bdb5.png)

Thoughts @sabernhardt?

Reply:
This was fixed in PR #3827 by separating the button from the input. Technically this is unresolved in this PR so I will leave it as such.